sklearn/utils/_bunch.py

```diff
@@ -0,0 +1,48 @@
+class Bunch(dict):
```

Comment:
Moved from `__init__.py` unchanged

Reply:
https://github.com/scikit-learn/scikit-learn/pull/22084 does this independently, merging that would remove this from the diff here.